static/src/js/components/GranuleResults/__tests__/GranuleResultsItem.test.js

```diff
@@ -335,316 +363,417 @@ function setup(type, overrideProps) {
+      setup('opensearch')
 
-      expect(enzymeWrapper.find('.granule-results-item__button--add').props().disabled).toBeTruthy()
+      expect(screen.getByRole('button', { name: 'Add granule' })).toBeDisabled()
     })
   })
 
-  describe('when clicking the remove button', () => {
+  describe('when clicking the Filter Granule button', () => {
     describe('with CMR granules', () => {
-      test('it removes the granule from results', () => {
-        const { enzymeWrapper, props } = setup('cmr')
-        const removeButton = enzymeWrapper.find(MoreActionsDropdownItem).at(1)
+      test('it removes the granule from results', async () => {
+        const { props, user } = setup('cmr')
+
+        await act(async () => {
+          const moreActions = screen.getByRole('button', { name: 'More actions' })
+          await user.click(moreActions)
+        })
+
+        await act(async () => {
+          await userEvent.click(screen.getByRole('button', { name: 'Filter granule' }), { pointerEventsCheck: 0 })
+        })
 
-        removeButton.simulate('click')
         expect(props.onExcludeGranule.mock.calls.length).toBe(1)
         expect(props.onExcludeGranule.mock.calls[0]).toEqual([{
           collectionId: 'collectionId',
           granuleId: 'granuleId'
         }])
-
-        expect(removeButton.props().title).toEqual('Filter granule')
       })
     })
 
     describe('with OpenSearch granules', () => {
-      test('it excludes the granule from results with a hashed granule id', () => {
-        const { enzymeWrapper, props } = setup('opensearch')
-        const removeButton = enzymeWrapper.find(MoreActionsDropdownItem).at(1)
+      test('it excludes the granule from results with a hashed granule id', async () => {
+        const { props, user } = setup('opensearch')
+
+        await act(async () => {
+          const moreActions = screen.getByRole('button', { name: 'More actions' })
+          await user.click(moreActions)
+        })
+
+        await act(async () => {
+          await userEvent.click(screen.getByRole('button', { name: 'Filter granule' }), { pointerEventsCheck: 0 })
+        })
 
-        removeButton.simulate('click')
         expect(props.onExcludeGranule.mock.calls.length).toBe(1)
         expect(props.onExcludeGranule.mock.calls[0]).toEqual([{
           collectionId: 'collectionId',
           granuleId: '170417722'
         }])
-
-        expect(removeButton.props().title).toEqual('Filter granule')
       })
     })
   })
 
   describe('when no granules are in the project', () => {
     test('does not have an emphisized or deepmphisized class', () => {
-      const { enzymeWrapper } = setup('cmr')
-      expect(enzymeWrapper.props().className).not.toContain('granule-results-item--emphisized')
-      expect(enzymeWrapper.props().className).not.toContain('granule-results-item--deemphisized')
+      setup('cmr')
+
+      const granuleResultsItem = screen.getByTestId('granule-results-item')
```

Comment:
Avoid using test ids, need to find a way to find this element in a way that better simulates how a user would use the app